content/introductions/reaping-rewards/index.tsx

```diff
@@ -6,14 +6,13 @@ export default function ReapingRewards({ lang }) {
 
   return (
     <Introduction lang={lang}>
-      <Text className="pt-2">{t('reaping_rewards.paragraph_one')}</Text>
-      <Text className="pt-2">{t('reaping_rewards.paragraph_two')}</Text>
+      <Text className="text-lg md:text-xl">{t('reaping_rewards.paragraph_one')}</Text>
+      <Text className="text-lg md:text-xl mt-4">{t('reaping_rewards.paragraph_two')}</Text>
+      <Text className="text-lg md:text-xl mt-4">{t('reaping_rewards.paragraph_three')}</Text>
     </Introduction>
   )
 }
 
 export const metadata = {
-  title: 'reaping_rewards.title',
```

Comment:
Sorry! i got a little hasty with removing everything here and the translation, It is still needed in the alt tag for the image.

Reply:
the subtitle however is not needed

Reply:
Got it, I made the change. Weird that the title is used as the alt text, maybe something we need to revise at some point.